test/controllers/AuthenticationControllerTest.java

```diff
@@ -45,19 +57,49 @@
     @Resource
     private TestUserAdminHelper helper;
     
+    @Resource
+    private StudyServiceImpl studyService;
+    
+    @Resource
+    private SchedulePlanServiceImpl schedulePlanService;
+    
     private TestUser testUser;
     
+    private SchedulePlan plan;
+    
+    private Study secondStudy;
+    
     @Before
     public void before() {
         testUser = helper.createUser(AuthenticationControllerTest.class);
+        
+        String id = RandomStringUtils.randomAlphabetic(7).toLowerCase();
+        secondStudy = new DynamoStudy();
+        secondStudy.setIdentifier(id);
+        secondStudy.setName("Second Test Study");
+        studyService.createStudy(secondStudy);
```

Comment:
@Before is run with every test. Not every test needs the second study. You should either change this to @BeforeClass or move it into the individual test method.